### ISSUE TITLE
Apple spell check

### ISSUE THREAD

**Issue by Parth** [lockbook/lockbook#753]
Does not spell check on old parts of the document, or copy and pasted parts of the document

**Comment by Parth** [comment]
Requested by @krishma-bawa

**Comment by smailbarkouch** [comment]
Implementing a [native cursor](https://developer.apple.com/documentation/appkit/text_display/adopting_the_system_text_cursor_in_custom_text_views) for macOS may lead to one solution. (it may integrate with [NSSpellChecker](https://developer.apple.com/documentation/appkit/nsspellchecker))

**Comment by Parth** [comment]
Yeah I personally would like some smart interactions with existing dictionaries on platforms where it makes sense. @smailbarkouch there's a bunch of intelligence suggestions built into iOS & macOS these days (tab to complete the rest of your sentence). Do you know if these APIs are exposed to us in any capacity?

**Comment by smailbarkouch** [comment]
Any plans for this

**Comment by Parth** [comment]
we talk about this pretty often, I too want this. You think it's top 3 for you?

**Comment by smailbarkouch** [comment]
Just seeing this - but I'd say so
